src/common/queries/useCustomNetworkApiInfo.ts

```diff
@@ -1,20 +1,13 @@
-import { UseQueryOptions, useQuery } from '@tanstack/react-query';
+import { useQuery } from '@tanstack/react-query';
 
-import { CoreNodeInfoResponse } from '@stacks/blockchain-api-client/src/generated/models';
-
-import { useApi } from '../api/useApi';
 import { DEFAULT_V2_INFO_ENDPOINT } from '../constants/constants';
 import { ONE_MINUTE } from './query-stale-time';
 
 export const getCustomNetworkApiInfo = (baseUrl: string) => () =>
   fetch(`${baseUrl}${DEFAULT_V2_INFO_ENDPOINT}`).then(res => res.json());
 
-export function useCustomNetworkApiInfo(
-  url: string,
-  options: Omit<UseQueryOptions<CoreNodeInfoResponse, Error>, 'queryKey'> = {}
-) {
-  const api = useApi();
-  return useQuery<CoreNodeInfoResponse, Error>({
+export function useCustomNetworkApiInfo(url: string, options: any = {}) {
+  return useQuery({
```

Comment:
I feel like this can be combined with useCoreAPIInfo

Reply:
for this PR I'm only performing a direct mapping to the new API client, so I'd prefer to refrain from introducing any additional refactors.